website/blog/2023-11-06-differentiate-prod-and-staging-environments.md

```diff
@@ -0,0 +1,73 @@
+---
+
+title: Why you need to differentiate your production and staging environments in dbt Cloud
```

Comment:
I like this title but also I feel like this post is _mostly_ about why you should set a production environment and doesn't hit on staging that much. Should we add more on staging environments and / or title something that really hammers this is about the importance of a prod environment?